pkg/controller/account/account_controller.go

```diff
@@ -203,6 +219,12 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 		return reconcile.Result{}, nil
 	}
 
+	awsSetupClient, err := r.awsClientBuilder.GetClient(controllerName, r.Client, setupClientInput)
```

Comment:
This line appears to reuse the `awsSetupClient` identifier which is used above in your PR on line 169. Is this a scoping issue where the earlier use on line 169 is meant to also be used here at this scope?

Reply:
If the awsSetupClient on line 169 is initialized there should be no reason the code execution for that same reconcile iteration should get here.  I can rename the client, though, to prevent this confusion.